ydb/docs/en/core/postgresql/statements/create_table.md

```diff
@@ -7,7 +7,7 @@ The `CREATE TABLE` statement is used to create an empty table in the current dat
 {% include [syntax.md](../../../_includes/postgresql/statements/create_table/syntax.md) %}
 
 When creating a table, you can specify:
-1. **Table Type**: `TEMPORARY` (not fully supported yet and used only in tests) – a temporary table that is automatically deleted at the end of the session or at the end of the current transaction. If this parameter is not set (left empty), a permanent table is created.
+1. **Table Type**: `TEMPORARY` / `TEMP` – a temporary table that is automatically deleted at the regular end of the session, otherwise it will no longer be available and will be automatically deleted after the time has elapsed. If this parameter is not set (left empty), a permanent table is created.
 2. **Table Name**: `<table name>` – you can use English letters in lowercase, numbers, and underscores. For example, the table name "People" will be stored as "people". For more information, see [Identifiers and Key Words](https://www.postgresql.org/docs/current/sql-syntax-lexical.html#SQL-SYNTAX-IDENTIFIERS).
```

Comment:
If you follow this link it says that way more characters are allowed and it's not clear what to trust: the description here or PostgreSQL's original documentation.